Few bug fixes

baatochan opened the pull request:
When testing v4.0.0 I found few small bugs that I patched.

1. When an event has 'Other' label (which you can select in Google Contacts - Birthday/Anniversary/Other) 'OTHER' is printed. I know this commit creates some problems with js-semistandard, but IMO it looks more organized that way. We can silence those warnings with comments, but I don't know how to write them.

2. When an event has a blank label (nothing selected, no custom text entered) 'null' is printed. I decided to put there 'Other' label, but I can change it to not print anything at all if that is a better solution.

3. Updated info about the maximum execution time of a single function. At the beginning I just put the info that @GioBonvi said in some comment, now I checked that in Google Script docs.

GioBonvi commented on code.gs:
```diff
@@ -561,12 +561,13 @@ Contact.prototype.getLines = function (type, date, format) {
     }
     // Custom label
     if (type === 'CUSTOM') {
+      var customEventLabel = event.getProp('label') || 'OTHER';
```
I'd move the variable declaration to line R359 to stick with the coding style (or get rid of the variable altogether and place `|| 'OTHER'` twice inline, but I prefer using the variable since it would be useful if more notification types are added in the future).

GioBonvi commented on code.gs:
```diff
@@ -1513,10 +1514,14 @@ function beautifyLabel (label) {
+    // Fake labels
     case 'OTHER_EMAIL': // Fake label for output.
-    case 'OTHER_PHONE': // Fake label for output,
+    case 'OTHER_PHONE': // Fake label for output.
```
I'd just move `OTHER_PHONE` and `OTHER_EMAIL` in the email and phone groups, since they already have inline comments explaining why they are different from the other ones.

Also I am not exactly sure what `// Offial Google label for events.` is supposed to mean: if you wanted to specify that it's not a fake label I'd remove it since the only two fake ones are well commented and all the other ones are "Official" by exclusion.

baatochan replied:
I forgot to remove this comment. I had put this before I grouped those labels.

baatochan commented:
I followed your guides, I guess I've corrected everything.

I also added '(falls through)' to silence linter. I know it looks kinda bad, but I have no other idea how to silence it.

GioBonvi commented on code.gs:
```diff
@@ -536,6 +536,7 @@ Contact.prototype.getLines = function (type, date, format) {
     }
     return typeMatch && event.getProp('day') === date.getDate() && event.getProp('month') === (date.getMonth() + 1);
   }).map(function (event) {
+    var eventLabel = event.getProp('label') || 'OTHER';
```
Sorry: I was not really clear before. What I meant was splitting decalration and assignment.

`var line, eventLabel, imgCount;` at line L539

`imgCountevent.getProp('label') || 'OTHER';` at line L564.

baatochan commented on code.gs:
```diff
@@ -1513,10 +1516,19 @@ function beautifyLabel (label) {
     case 'PAGER':
+    case 'OTHER_PHONE': // Fake label for output.
+    /*
+     * (falls through)
```
I would suggest leaving 'falls through' in brackets as the main reason for these comments is to group labels, not about 'falls through'.
